Octokit/Models/Response/CheckRunAnnotation.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CheckRunAnnotation
+    {
+        public CheckRunAnnotation()
+        {
+        }
+
+        public CheckRunAnnotation(string filename, string blobHref, int startLine, int endLine, CheckWarningLevel warningLevel, string message, string title, string rawDetails)
```

Comment:
Missing a property for `Title`.

Reply:
The response payload actually doesn't have that field, only the request seems to

Reply:
Odd, okay..

Reply:
after checking internally, turns out this was an oversight and `title` will be added to this response, so I've added it to the mode :+1:  Also I realised I still had it in the `ctor` anyway, even though the class didn't have the property, which is probably what you were pointing out anyway!  🤣